hoard/models.py

```diff
@@ -1,16 +1,139 @@
+from typing import Any, Dict, List, Optional
+
 import attr
 
 
+@attr.s(auto_attribs=True)
+class Author:
+    authorName: str
+    authorAffiliation: str
+    authorIdentifierScheme: Optional[str] = None
+    authorIdentifierValue: Optional[str] = None
+
+
+@attr.s(auto_attribs=True)
+class Contact:
+    datasetContactName: str
+    datasetContactEmail: str
+    datasetContactAffiliation: Optional[str] = None
+
+
+@attr.s(auto_attribs=True)
+class Description:
+    dsDescriptionValue: str
+    dsDescriptionDate: Optional[str] = None
+
+
 @attr.s(auto_attribs=True)
 class Dataset:
+    authors: List[Author]
+    contacts: List[Contact]
+    description: List[Description]
+    subjects: List[str]
     title: str
 
     def asdict(self) -> dict:
         return attr.asdict(self)
 
+    def dv_format(self) -> dict:
```

Comment:
I think I would either delete `asdict()` or just move the body of this function into that function. The canonical dictionary representation of a Dataset should probably just be the Dataverse format.

Reply:
Done 👍